incubator/elasticsearch/values.yaml

```diff
@@ -11,6 +11,10 @@ image:
   # pullSecrets:
   #   - registry-secret
 
+busybox:
```

Comment:
I know it wasn't there before, but please add a tag and pullpolicy for the busybox image based on best practices.

Reply:
No problem, will do it

Reply:
@paulczar should i use Always pull policy when the tag is latest?

Reply:
@paulczar i think everything is ok now, can you valid the changes?